cmd/controller/controller.go

```diff
@@ -67,6 +67,11 @@ func AddConfigFlags(cmd *cobra.Command) {
 		10*time.Minute,
 		"time to retain kubernetes jobs after completion",
 	)
+	cmd.Flags().Duration(
+		"poll-interval",
+		time.Second,
+		"time to wait between polling for new jobs",
```

Comment:
I think it's important to call out the effect on speed to pick up jobs:

```suggestion
		"time to wait between polling for new jobs (minimum 1s); note that increasing this causes jobs to be slower to start",
```